lib/WebService/MinFraud/Record/Device.pm

```diff
@@ -74,6 +86,14 @@ This is the date and time of the last sighting of the device on the specified
 IP address for your user account. The string is formatted in the ISO 8601
 combined date and time in UTC.
 
+=head2 session_age
+
+A session length (in seconds) consisting of a floating point number.
```

Comment:
I wonder if we could provide a bit more detail on what this actually is supposed to be as I would not be sure as an implementer and would likely supply the wrong value because of that.

I _think_ is is supposed to be the number of seconds between when the user's session was first created and the time of the transaction. It should not the duration of the current visit but should be the time since the start of the first visit.